src/library_gl.js

```diff
@@ -1400,6 +1411,97 @@ var LibraryGL = {
     GLctx.bufferSubData(target, offset, HEAPU8.subarray(data, data+size));
   },
 
+  // Queries EXT
+  glGenQueriesEXT__sig: 'vii',
+  glGenQueriesEXT: function(n, ids) {
+    for (var i = 0; i < n; i++) {
+      var query = GLctx['createQueryEXT']();
```

Comment:
Let's call `context.disjointTimerQueryExt.createQueryEXT()` here directly now that we don't do the routing stubs.